src/__tests__/lib/exampleData.js

```diff
@@ -244,6 +272,56 @@ export const streamMessage = (extra?: $Rest<Message, {}>): Message => {
   return deepFreeze({ ...baseMessage, ...extra });
 };
 
+/* ========================================================================
+ * Outbox messages
+ *
+ * (Only stream messages for now. Feel free to add PMs, if you need them.)
+ */
+
+/** An outbox message with no interesting data. */
+const outboxMessageBase: $Diff<Outbox, {| id: mixed, timestamp: mixed |}> = deepFreeze({
+  isOutbox: true,
+  isSent: false,
+
+  avatar_url: 'https://example.com/avatar.png',
+  content: '<p>Test.</p>',
+  display_recipient: 'test',
+  // id: ...,
+  markdownContent: 'Test.',
+  narrow: [{ operator: 'stream', operand: 'test' }],
+  reactions: [],
+
+  subject: 'test topic',
+  sender_email: selfUser.email,
+  sender_full_name: selfUser.full_name,
+  // timestamp: ...,
+  type: 'stream',
+});
+
+/** Create an outbox message from an interesting subset of its data. */
+export const makeOutboxMessage = (data: $Shape<Outbox>): Outbox => {
+  // In real data, the .id and .timestamp of an Outbox object are identical.
+  // Complain if a test tries to violate that constraint.
+  const { id, timestamp } = data;
+  if (id !== undefined && timestamp !== undefined && id !== timestamp) {
+    throw new Error(
+      `outbox messages should have identical .id ${id} and .timestamp (${timestamp})`,
+    );
+  }
+
+  const outputTimestamp = id ?? timestamp ?? makeTime() / 1000;
```

Comment:
Given this, there's no use case for the test to choose one or the other to set. Keep the contract and the implementation simpler by just disallowing `id`.

Reply:
Looks good but I think the implementation comment
```
  // In real data, the .id and .timestamp of an Outbox object are identical.
  // Complain if a test tries to violate that constraint.
```
no longer applies and can be deleted too.